app/models/course/requirement.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Requirement < ActiveRecord::Base
+  actable
+  belongs_to :has_requirement, polymorphic: true # naming is hard
```

Comment:
requirable  ? Seems the convention of polymorphic naming is "xxxable".

http://guides.rubyonrails.org/association_basics.html#polymorphic-associations